accepted/00000-hub-reporting.md

```diff
@@ -70,26 +56,58 @@ from the main Uyuni Server database.
 Data which belong to the traditional stack only (like osad status) should not be made available in the
 reporting database as the traditional stack will be deprecated soon.
 
+# Installation
+The reporting tools will be installed by default; all the upgrade to a newer version, will install the 
+reporting tools as well.
 
-## Adding Data to the Reporting DB
+We provide a new package installed in the hub system:
+- to setup the reporting DB schema,
+- to setup and manage the reporting database,
+- to setup and manage DBs in all Uyuni Server.
```

Comment:
Is this only for the Hub ? I think this is generic as we try to keep the reportig DB on a single server compatible with the reporting DB on the Hub.